src/game/client/components/chat.cpp

```diff
@@ -551,7 +551,7 @@ void CChat::OnMessage(int MsgType, void *pRawMsg)
 		if(!m_CommandManager.AddCommand(pMsg->m_Name, pMsg->m_HelpText, pMsg->m_ArgsFormat, ServerCommandCallback, this))
 			dbg_msg("chat_commands", "adding server chat command: name='%s' args='%s' help='%s'", pMsg->m_Name, pMsg->m_ArgsFormat, pMsg->m_HelpText);
```

Comment:
here it says chat_commands, so this is inconsistent in it's current implmenetation